packages/typeorm/src/mixins/typeorm.mixin.ts

```
@@ -0,0 +1,64 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/typeorm
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, MixinTarget} from '@loopback/core';
+import debugFactory from 'debug';
+import {ConnectionOptions} from 'typeorm';
+import {TypeOrmComponent} from '../';
+
+const debug = debugFactory('loopback:typeorm:mixin');
+
+export function TypeOrmMixin<T extends MixinTarget<Application>>(
+  superClass: T,
+) {
+  return class extends superClass {
+    typeormConnectionOptions: ConnectionOptions[] = [];
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    constructor(...args: any[]) {
+      super(...args);
+      this.component(TypeOrmComponent);
+    }
+
+    async migrateSchema(options: SchemaMigrationOptions = {}): Promise<void> {
+      // TODO: implement using TypeORM
```

Please throw an error instead of skipping the migration silently.